config/kraft/server.properties

```diff
@@ -31,22 +31,23 @@ controller.quorum.voters=1@localhost:9093
 
 ############################# Socket Server Settings #############################
 
-# The address the socket server listens on. It will get the value returned from
-# java.net.InetAddress.getCanonicalHostName() if not configured.
+# The address the socket server listens on. If not configured, the host name will be equal to the value of
+# java.net.InetAddress.getCanonicalHostName(), with PLAINTEXT listener name, and port 9092.
 #   FORMAT:
 #     listeners = listener_name://host_name:port
 #   EXAMPLE:
 #     listeners = PLAINTEXT://your.host.name:9092
 listeners=PLAINTEXT://:9092,CONTROLLER://:9093
+
+# Name of listener used for communication between brokers.
 inter.broker.listener.name=PLAINTEXT
 
-# Hostname and port the broker will advertise to producers and consumers. If not set,
-# it uses the value for "listeners" if configured.  Otherwise, it will use the value
-# returned from java.net.InetAddress.getCanonicalHostName().
+# Listener name, hostname and port the broker will advertise to producers and consumers.
+# If not set, it uses the value for "listeners".
 advertised.listeners=PLAINTEXT://localhost:9092
 
-# Listener, host name, and port for the controller to advertise to the brokers. If
-# this server is a controller, this listener must be configured.
+# A comma-separated list of the names of the listeners used by the controller.
+# This is required if running in KRaft mode.
```

Comment:
It might be worth pointing out how that the controller listener is assumed to be PLAINTEXT as long as all other listeners are configured for PLAINTEXT as well.

Reply:
Added one more line:
# A comma-separated list of the names of the listeners used by the controller.
**# If no explicit mapping set in `listener.security.protocol.map`, default will be using PLAINTEXT protocol**
# This is required if running in KRaft mode.